releng/org.eclipse.xtext.tycho.parent/pom.xml

```diff
@@ -25,8 +25,8 @@
 		<log4j.version>1.2.16</log4j.version>
 		<commons-logging.version>1.1.3</commons-logging.version>
 		<commons-cli.version>1.2</commons-cli.version>
-		<equinox-common-version>3.6.0</equinox-common-version>
-		<jdt-core-version>3.10.0</jdt-core-version>
+		<equinox-common-version>3.8.0</equinox-common-version>
+		<jdt-core-version>3.12.2</jdt-core-version>
```

Comment:
There are still Maven builds without Tycho in xtext-maven and xtext-xtend. There the properties are still relevant. But for Tycho builds the dependencies are determined via each bundle's MANIFEST.MF and the common target platform.